src/TailscaleManager.hs

```diff
@@ -1,4 +1,4 @@
--- | Tailscale Routes Manager
+{-# LANGUAGE QuasiQuotes #-}
```

Comment:
I think you can omit this directive, because `QuasiQuotes` is included in the cabal `default-extensions` list for this library.

Reply:
Yeah sure, resolved :)